### PR TITLE
update to paper-styles#0.8.2

### PR DESCRIPTION
**Pull request by morethanreal**
@cdata PTAL

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -22,8 +22,7 @@
   "dependencies": {
     "polymer": "Polymer/polymer#0.8-preview",
     "core-selector": "Polymer/core-selector#0.8-preview",
-    "layout": "Polymer/layout",
-    "paper-styles": "PolymerLabs/paper-styles"
+    "paper-styles": "PolymerElements/paper-styles#0.8.2"
```
Comment (cdata) [inline bower.json]: Why not `^0.8.0`? Are you worried about breaking API changes before we hit `0.9.0`?

Comment (morethanreal) [inline bower.json]: I guess we shouldn't worry about it. Will fix.

Comment (nevir) [inline bower.json]: Yeah, we should be in the habit of always doing `^X.Y.Z` unless it's a `-rc.#` style tag

Comment (nevir) [inline bower.json]: Also, I'm a jerk and just pushed out `0.8.4` of paper-styles (bump polymer to `#v0.8.0-rc.4` too)

Comment (morethanreal) [inline bower.json]: Hah! But I changed it to `^0.8.0`!

File: bower.json [hunk 2]
```diff
@@ -31,5 +29,8 @@
     "paper-item": "PolymerElements/paper-item#master",
     "test-fixture": "Polymer/test-fixture#master",
     "web-component-tester": "Polymer/web-component-tester#^2.2.0"
+  },
+  "resolutions": {
+    "polymer": "v0.8.0-rc.4"
   }
```
Comment (nevir) [inline bower.json:35]: We should avoid checking in resolutions for element repos

Comment (morethanreal) [inline bower.json:35]: How do I handle my dependencies having conflicting versions of a package? I don't think I should just let the tests fail.

**Pull request merged**